pkg/controllers/internalmembercluster/v1beta1/member_controller.go

```diff
@@ -196,7 +196,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	var imc clusterv1beta1.InternalMemberCluster
 	if err := r.hubClient.Get(ctx, req.NamespacedName, &imc); err != nil {
-		klog.ErrorS(err, "Failed to get internal member cluster: %s", req.NamespacedName)
+		klog.ErrorS(err, "Failed to get internal member cluster", "InternalMemberCluster", req.NamespacedName)
```

Comment:
```suggestion
		klog.ErrorS(err, "Failed to get internal member cluster", "internalMemberCluster", req.NamespacedName)
```
need to use lower cameral case 
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments

Reply:
The code was added long time ago, we need to fix the above too :(

Reply:
fixed.